src/resources/wait-time.ts

```diff
@@ -0,0 +1,68 @@
+import { AxiosInstance } from 'axios';
+
+import Conditional from './conditional';
+
+export interface WaitTimeParameters {
+  location_id?: string | number;
+  limit?: number;
+  page?: number;
+}
+
+export interface WaitTimeResource {
+    on(page: number): this;
+    take(limit: number): this;
+    at(location_id: string | number): this;
+}
+
+export default class WaitTime extends Conditional implements WaitTimeResource {
+  protected client: AxiosInstance;
+  protected location_id: string | number | null;
+  protected page: number | null;
+  protected limit: number | null;
+
+  constructor(client: AxiosInstance) {
+    super();
+
+    this.client = client;
+    this.location_id = null;
+    this.page = null;
+    this.limit = null;
+  }
+
+  public async get(): Promise<any> {
+    let params: WaitTimeParameters = {};
+    let location_id: string | number = '';
+
+    if (this.location_id != null) {
+        location_id = this.location_id;
+    }
+
+    if (this.limit) {
+      params.limit = this.limit;
+    }
+
+    if (this.page) {
+      params.page = this.page;
+    }
+
+    return await this.client.get(`wait-time-average/${location_id}`.replace(/\/$/, ''), { params });
```

Comment:
What is the purpose of the regex here?

Reply:
just removes the slash at the end, if there is one

Reply:
Is it actually required? I'm not sure why that would need to be in there

Reply:
definitely not required, i just didn't like seeing the trailing slash haha. i can remove it.